### PR TITLE
Remove Knative Serving Charts

### PR DESCRIPTION
**Pull request by anishj0shi**
Remove knative-serving chart and knative-serving test

other references to knative-serving will be tackled with different PRs

### CODE REVIEW

File: resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml [hunk 1]
```diff
@@ -1,31 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  annotations:
-    "helm.sh/resource-policy": keep
```
Comment (magicmatatjahu) [inline resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml:5]: `"helm.sh/resource-policy": keep` annotation tells Helm that resource should stay on cluster, but it's will unpinned from Helm release, so you should create upgrade job and remove whole crds related to `knative-serving`.

Comment (k15r) [reply resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml:5]: we will create documentation how to cleanup the cluster after knative serving was removed. This guarantees that customers who used knative serving for their own functionality will not lose any configuration.

**Review by alexandra-simeonova — APPROVED** [review]

**Review by Demonsthere — DISMISSED** [review]
Please keep in mind that deleting helm resources this way introduces a problem: The SA you created for the job will persist the installation (as it does not have a succeed phase for the deletion hook), which someone could use to delete arbitrary helm releases. 
@pbochynski Your insight might be needed here

**Review by sayanh — APPROVED** [review]

File: resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml (hunk 1 above)
Comment (nachtmaar) [reply resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml:5]: Migration guide can be found here: https://github.com/kyma-project/kyma/pull/9487#pullrequestreview-493484269

**Review by marcobebway — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Review by pbochynski — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Review by piotrmsc — APPROVED** [review]

**Pull request merged**